core/build.gradle.kts

```diff
@@ -9,9 +9,14 @@ tasks.withType<KotlinCompile>().configureEach {
     kotlinOptions.allWarningsAsErrors = true
 }
 
+rootProject.plugins.withType<org.jetbrains.kotlin.gradle.targets.js.nodejs.NodeJsRootPlugin> {
+    rootProject.the<org.jetbrains.kotlin.gradle.targets.js.nodejs.NodeJsRootExtension>().nodeVersion = "16.0.0"
+}
```

Comment:
I just used solution suggested in issue, I decided not to experiment first of all because I have no MacArm64 machine in directly access, only on TC. And the second point I hope JS team fix this issue soon and we'll be able to remove this workaround.